src/shared_modules/utils/registryHelper.h

```diff
@@ -195,6 +242,35 @@ namespace Utils
                 return std::string{reinterpret_cast<const char*>(spBuff.get())};
             }
 
+
+            bool longint(const std::string& valueName, ULONGLONG& value) const
```

Comment:
could you rename to qword ?

```suggestion
            bool qword(const std::string& valueName, ULONGLONG& value) const
```